bindings/rascal/models/kernels.py

```diff
@@ -0,0 +1,44 @@
+# from ..lib.Kernels import CosineKernel
+from ..lib._rascal.Models.Kernels import Kernel as Kernelcpp
+from ..neighbourlist import AtomsList
+import json
+
+class Kernel(object):
+    """
+    Computes the kernel from an representation
+
+    Attributes
+    ----------
+
+    Methods
+    -------
+    """
+    def __init__(self, representation, name='Cosine', target_type='structure', **kwargs):
```

Comment:
Parameters need documenting, either in the class or the (currently missing) `__init__` docstring